.gitignore

```diff
@@ -1,5 +1,6 @@
 build/
+log/
 
 # IDE
 .vscode/
-.cache/
\ No newline at end of file
+.cache/
```

Comment:
Was there a reason to change this file? I would propose to leave it out of the diff

Reply:
Ah, I see, in that case could you please remove the changes to the `.gitignore` file? Then I can merge the PR.

Reply:
yes of course